homeassistant/components/decora_wifi/light.py

```diff
@@ -114,10 +59,10 @@ def turn_on(self, **kwargs):
         attribs = {"power": "ON"}
 
         if ATTR_BRIGHTNESS in kwargs:
-            min_level = self._switch.data.get("minLevel", 0)
-            max_level = self._switch.data.get("maxLevel", 100)
-            brightness = int(kwargs[ATTR_BRIGHTNESS] * max_level / 255)
-            brightness = max(brightness, min_level)
+            maxlevel = self._switch.data.get("maxLevel", 100)
+            minlevel = self._switch.data.get("minLevel", 100)
```

Comment:
```suggestion
            max_level = self._switch.data.get("maxLevel", 100)
            min_level = self._switch.data.get("minLevel", 100)
```

Please separate words with `_`